src/config.h

```diff
@@ -175,6 +175,9 @@ namespace config {
     std::string file_state;
 
     std::string external_ip;
+    std::vector<std::string> resolutions;
+    std::vector<int> fps;
+    int max_bitrate;
```

Comment:
```suggestion
    int max_bitrate;
```

These two settings have been removed from Sunshine.